src/robusta/integrations/slack/sender.py

```diff
@@ -235,7 +262,8 @@ def prepare_slack_text(self, message: str, max_log_file_limit_kb: int, files: Li
                 if len(file_block.contents) == 0:
                     continue
                 permalink = self.__upload_file_to_slack(file_block, max_log_file_limit_kb=max_log_file_limit_kb)
-                uploaded_files.append(f"* <{permalink} | {file_block.filename}>")
+                if permalink:
```

Comment:
Don't we want to show message in the slack that file is missing due to error?

Reply:
what do you have in mind?

Reply:
In my opinion, having the error logged is sufficient.
If someone looks for the file and doesn't see it, they'll naturally check the runner logs, find the error, and be able to fix the issue.

Adding an error message directly into the Slack message would make it look messy and could give the impression that the system is unstable.

Reply:
added message according to thread